capstone-rs/src/instruction.rs

```diff
@@ -145,6 +145,14 @@ pub struct Insn<'a> {
 /// `ArchDetail` enum.
 pub struct InsnDetail<'a>(pub(crate) &'a cs_detail, pub(crate) Arch);
 
+/// Contains information about registers accessed by an instruction, either explicitly or implicitly
+pub struct InsnRegsAccess {
```

Comment:
Add `#[derive(Debug, Clone, PartialEq, Eq)]`

Reply:
`Debug, PartialEq, Eq` can't be derived because of the array with size `>32`, I added a comment

Reply:
Could you manually impl `PartialEq`, `Eq`?